.github/workflows/elixir-ci.yml

```diff
@@ -13,15 +13,8 @@ on:
       - 'native/**'
       - 'priv/**'
       - 'test/**'
-      - '!.github/**'
```

Comment:
reason for this is because doesn't make sense to re-run elixir ci when non-elixir code changes, reduces iteration time when working on ci/cd related or docs stuff.
maybe a better glob pattern would work

Reply:
That doesn't matter, as all these entries were doing nothing anyway. These filters works that this action will be fires only if any files matches paths defined there. So negative matches that aren't within positive matches aren't needed at all.